### PR TITLE
Blocking all update edges from ocp 4.8 to 4.9

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: blocked-edges/4.9.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+to: 4\.9\..*
```
Comment (wking) [inline blocked-edges/4.9.yaml:1]: [`to` doesn't support regular expressions][1] (only [`from` does][2]), so you'll want to iterate over all 4.9.z and create a copy of this that blocks updates from 4.8.  Amazingly, this is our first to-4.9 block, so we don't need to worry about merging with existing to-4.9 issues.  You can just iterate over all the 4.9.z in `fast-4.9`.

[1]: https://github.com/openshift/cincinnati-graph-data/blame/d8c05513732b51a2a49735497609bcd0c945c1a2/README.md#L102
[2]: https://github.com/openshift/cincinnati-graph-data/blame/d8c05513732b51a2a49735497609bcd0c945c1a2/README.md#L103

**Review by wking — APPROVED** [review]
/lgtm

**Comment by davidkarlsen** [comment]
Ouch, https://bugzilla.redhat.com/show_bug.cgi?id=2068601 is not public, but we've upgraded from 4.8 to 4.9 - do we risk any etcd problems?

**Review by sdodson — CHANGES_REQUESTED** [review]

File: build-suggestions/4.9.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # If you specify an arch it must have the full set of values
 ---
 default:
-  minor_min: 4.8.14
+  minor_min: 4.8.9999
```
[the remark this answers is not in the document]
Comment (LalatenduMohanty) [reply build-suggestions/4.9.yaml]: Fixed

**Comment by LalatenduMohanty** [comment]
> Ouch, https://bugzilla.redhat.com/show_bug.cgi?id=2068601 is not public, but we've upgraded from 4.8 to 4.9 - do we risk any etcd problems?

We are trying to figure these things out and will make the bug public soon

**Comment by wking** [comment]
> https://bugzilla.redhat.com/show_bug.cgi?id=2068601 is not public...

It is now :)

**Pull request merged**